packs/bestiary-effects/effect-demon-touched-weapon.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_id": "3PiE6Wka0ylO5V1x",
+    "img": "icons/magic/unholy/hand-weapon-glow-black-green.webp",
+    "name": "Effect: Demon-Touched Weapon",
+    "system": {
+        "description": {
+            "value": "<p>Shenga deals an additional 1d6 bleed damage with her melee Strikes.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Society Scenario #5-06: Ukuja, the First Wall"
+        },
+        "rules": [
+            {
+                "damageType": "bleed",
+                "diceNumber": 1,
+                "dieSize": "d6",
+                "key": "DamageDice",
```

Comment:
May want:
```
"predicate": [
        "melee"
],
```

Reply:
She only has melee strikes but it is theoretically faster to restrict by selector `melee-strike-damage` than predicate, since both get tested against and selector goes first (not that you would ever notice the difference). Still, added the selector change.